microservices/src/main/java/io/silverware/microservices/Boot.java

```diff
@@ -47,6 +47,7 @@
 public final class Boot {
 
    private static final Logger log = LogManager.getLogger(Boot.class);
+   static boolean IS_TESTING = false;
```

Comment:
I do not think this is a good solution. What about using a system property instead? Something like `silverware.system.exit` which would be set to `true` by default.